packages/react/src/themes/teams/components/TextArea/textAreaVariables.ts

```diff
@@ -1,11 +1,41 @@
-import { ResizeProperty } from 'csstype'
+import { pxToRem } from '../../../../lib'
 
 export interface TextAreaVariables {
+  backgroundColor: string
+  invertedBackgroundColor: string
+  disabledBackgroundColor: string
+  disabledColor: string
+  borderColor: string
+  borderRadius: string
+  borderWidth: string
+  fontColor: string
+  fontSize: string
+  borderColorFocus: string
+  placeholderColor: string
   margin: string
-  resize: ResizeProperty
+  padding: string
+  height: string
 }
 
-export default (siteVars): TextAreaVariables => ({
+export default (siteVars): Partial<TextAreaVariables> => ({
```

Comment:
```suggestion
export default (siteVars): TextAreaVariables => ({
```

I am not sure that it should be `Partial` as we don't have base theme anymore.